RecoBTag/FeatureTools/interface/deep_helpers.h

```diff
@@ -102,5 +102,17 @@ namespace btagbtvdeep {
     VarInfo info(const std::string &name) const { return var_info_map.at(name); }
   };
 
+  int center_norm_pad(const std::vector<float> &input,
+                      float center,
+		      float scale,
```

Comment:
before submitting the official PR, you should run `scram b code-checks` and `scram b code-format` (then examine, compile, test, and commit the results). This branch now has a mix of tabs and spaces for indentation, which makes it hard to read (and also is not allowed by code-format).

Reply:
Done for the most recent commit